traffic_portal/app/src/common/modules/table/serverServerCapabilities/table.serverServerCapabilities.tpl.html

```diff
@@ -0,0 +1,53 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div class="x_panel">
+    <div class="x_title">
+        <ol class="breadcrumb pull-left">
+            <li><a ng-click="navigateToPath('/servers')">Servers</a></li>
+            <li><a ng-click="navigateToPath('/servers/' + server.id)">{{::server.hostName}}</a></li>
+            <li class="active">Capabilities</li>
+        </ol>
+        <div class="pull-right">
+            <button name="addServerCapabilityButton" class="btn btn-primary" title="Add Server Capability" ng-click="addServerCapability()"><i class="fa fa-plus"></i></button>
+            <button class="btn btn-default" title="Refresh" ng-click="refresh()"><i class="fa fa-refresh"></i></button>
```

Comment:
Both of these are `button[type="submit"]`, which I think is probably not appropriate.